roles/kafka/README.rst

```diff
@@ -0,0 +1,107 @@
+Kafka
+======
+
+.. versionadded:: 1.1
+
+This role installs the `Kafka Mesos Framework <https://github.com/mesos/kafka>`_
+and starts Kafka brokers.
+
+Installation
+------------
+
+After a successful initial run (from your customized ``sample.yml``), you can
+install Kafka with ``ansible-playbook -e @security.yml addons/kafka.yml``. It
+can take several minutes for all components to deploy and become healthy.
+
+Accessing the Kafka Mesos REST API
+----------------------------------
+
+After the Kafka framework and the Kafka brokers have been successfully started
+and initialized, it should be possible to access the Kafka Mesos REST API at
+``/kafka`` on control nodes.
+
+Default Configuration
+---------------------
+
+The default configuration of the Kafka brokers will require at least 3 worker
+nodes that each have at least 4 CPUs and 4 GBs of memory available to Mesos.
+
+Depending on your planned environment, you may wish to customize the sizing of
+your Kafka cluster using the variables documented below.
+
+Customizing your Installation
+-----------------------------
+
+The size of your Kafka cluster is controlled by the variables documented below.
+
+Variables
+---------
+
+.. data:: kafka_scheduler_name
+
+   The application ID of the Kafka scheduler in Marathon.
+
+   default: "mantl/kafka"
+
+.. data:: kafka_service_name
+
+   The name of the service that is registered in Consul when the framework is
+   deployed. This needs to match what would be derived via mesos-consul. For
+   example, when ``kafka_scheduler_name`` is set to ``mantl/kafka``, the service
+   name should be ``kafka-mantl``.
+
+   default: "kafka-mantl"
+
+.. data:: kafka_scheduler_cpu
+
+   The amount of CPU to allocate to the Kafka scheduler instance (MB).
+
+   default: 0.2
+
+.. data:: kafka_scheduler_mem
+
+   The amount of memory to allocate to the Kafka scheduler instance (MB).
+
+   default: 512
+
+.. data:: kafka_broker_count
+
+   The number of Kafka brokers to start.
+
+   default: 3
+
+.. data:: kafka_broker_cpu
```

Comment:
too many CPUs for only 4GB ram,
lets follow the 1/4 proportion (1cpu/4gb, 2cpu/8gb, ...)